### DISCUSSION TITLE
Basic avitab tablet intagration

### DISCUSSION THREAD

**Discussion by ghost** [magknight/787aviatorsTracker#524] [Completed things] [answered]
Add the tablet above the default efb with the map on it

**Comment by aeroniemi** [comment]
Dependent on:
modelling - design of a tablet
settingsCore - option to have it enabled globally, default off
graphics - see if it's possible to make the interface look more 787y

**Comment by ghost** [comment]
there is a tablet that comes default with the avitab plugin you can
integrate

**Comment by Simleybob** [comment]
This would be good, like in the real aircraft (depends on airline ofc): 
![image](https://user-images.githubusercontent.com/81443205/114752993-da469200-9d4e-11eb-9025-8e359d6b54d9.png)

**Comment by bturtle** [comment]
I agree. We should integrate it.

**Comment by Simleybob** [comment]
it has now been completed - added in 1.7.12

**Comment by aeroniemi** [comment, marked as answer]
Has been added as part of v1.7.12
